src/main/java/mate/academy/Main.java

```diff
@@ -1,20 +1,40 @@
+/**
+ * Contains the main class of the application.
+ */
 package mate.academy;
 
-import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
 import mate.academy.service.AuthenticationService;
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
 
-public class Main {
-    public static void main(String[] args) {
-        AuthenticationService authenticationService = new AuthenticationServiceImpl();
+import java.util.logging.Logger;
+
+public final class Main {
+    /**
+     * The logger for the Main class.
+     */
+    private static Logger logger = Logger.getLogger(
+            Main.class.getName());
```

Comment:
make logger the private static final. 
getLogger() can accept the Main.class directly
```suggestion
    private static final Logger logger = Logger.getLogger(
            Main.class);
```

Reply:
[ERROR] C:\Users\User\IdeaProjects\jv-logger\src\main\java\mate\academy\Main.java:18:33: Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [ConstantName]
Audit done.
when i do it final it asks me to do constant, and when i do constant other mentor is saying to do it in lowercase, and what shal i  do?))) and when i do it like on a example i must wrap it in toString, and i still have the same error